sources/packages/api/src/route-controllers/education-program-offering/education-program-offering.aest.controller.ts

```diff
@@ -71,4 +76,36 @@ export class EducationProgramOfferingAESTController extends BaseController {
       payload.offeringStatus,
     );
   }
+  /**
+   * Get all offerings that were requested for change and waiting to be approved/declined.
+   * @returns offerings.
```

Comment:
can you update something like `list of offerings that were requested for change.`